### PR TITLE
Getting started wording and docs organization updates.

### PR DESCRIPTION
**Pull request by dhatch**
- Remove old roles guides.
- Add home link.
- Move write policies to how to.
- Update wording on get started section index page.
- Update what's next section of quickstart.

### CODE REVIEW

File: docs/content/any/guides/policies/index.md [hunk 1]
```diff
@@ -144,7 +145,7 @@ let's take a moment to put them in context. Rules are loaded into
 a **knowledge base**, a kind of specialized in-memory database that
 supports **queries** by pattern matching and logical inference.
 
-![Oso Architecture](getting-started/policies/arch.svg)
+![Oso Architecture](guides/policies/arch.svg)
```
Comment (gj) [inline docs/content/any/guides/policies/index.md:148]: is there any difference between linking to `guides/policies/arch.svg` vs. `/guides/rbac`? (note the leading slash)

File: docs/content/any/project/changelogs/2021-06-09.md [hunk 2]
```diff
@@ -55,8 +55,6 @@ We have renamed this field to `permissions` for clarity.
 
 Previously, we required role names to be globally unique. Now, role names will be internally namespaced, removing the globally unique requirement. Like permissions, the role namespace is the resource name specified in the `resource` predicate. Roles names must be unique within a single resource namespace. Roles associated with other resources must be referenced using the namespace. Roles within the same resource can be referenced without the namespace.
 
-Below is an example `resource` predicate that reflects the above changes. For a more in-depth example, see our [documentation]({{< ref path="/guides/roles/sqlalchemy" lang="python" >}}) and [example repository](https://github.com/osohq/gitclub/tree/main/backends/flask-sqlalchemy).
```
Comment (gj) [inline docs/content/any/project/changelogs/2021-06-09.md:58]: why delete this whole bullet but leave the snippet below, now w/ less context?

File: docs/content/python/guides/ui/index.md [hunk 3]
```diff
@@ -135,8 +135,7 @@ allowed to take more actions on the repository than Sully.
 With this relatively straightforward policy, it's easy to trace where
 the users' allowed actions come from. But `Oso.get_allowed_actions()`
 can be especially powerful with more complicated policies. For example,
-if we used Oso's [SQLAlchemy Roles library
-features](guides/roles/sqlalchemy),
+if we used Oso's [roles](guides/rbac),
 we could have a policy that looks like this instead:
```
Comment (gj) [inline docs/content/python/guides/ui/index.md:139]: this isn't accurate

File: docs/content/any/getting-started/_index.md [hunk 4]
```diff
@@ -1,11 +1,44 @@
...
+# Get Started
 
-Learn the basics of using Oso and get Oso set up in your application.
+Let's get started with Oso! Oso is a framework for application
+authorization with built-in primitives for modeling authorization. Oso
+consists of the following components:
+
+- **Oso Library**: Oso is a library that you include in your application to
+enforce authorization. The library supports multiple languages,
+currently Python, Node, Ruby, Go, Java, and Rust.
```
Comment (gj) [inline docs/content/any/getting-started/_index.md:15]:
```suggestion
currently Python, Node.js, Ruby, Go, Java, and Rust.
```

File: docs/content/any/getting-started/_index.md [hunk 5]
```diff
@@ -1,11 +1,44 @@
...
+- **Framework Libraries**: Oso has pre-built integrations for web
+application libraries, including Django, SQLAlchemy, and Flask.
+{{% /ifLang %}}
+
+To quickly have a runnable example of using Oso, check out our
+[Quickstart](quickstart) where you can see a sample app with Oso.
+
+To use Oso for authorization in an existing application, check out our
+[Add Oso to Your App guide](application), where you'll be introduced to
+enforcing authorization, setting up role based authorization, writing
```
Comment (gj) [inline docs/content/any/getting-started/_index.md:38]:
```suggestion
enforcing authorization, setting up role-based authorization, writing
```

File: docs/content/any/getting-started/_index.md [hunk 6]
```diff
@@ -1,11 +1,44 @@
...
+rules, and filtering data by authorization.
```
Comment (gj) [inline docs/content/any/getting-started/_index.md:39]: not sure "filtering data by authorization" is descriptive enough

```suggestion
rules, and filtering large collections to only contain authorized elements.
```

File: docs/content/any/getting-started/_index.md [hunk 7]
```diff
@@ -1,11 +1,44 @@
...
+
+For more on the design principles behind Oso and why it makes
+authorization easy for development teams see the **TODO** What is Oso
```
Comment (gj) [inline docs/content/any/getting-started/_index.md:42]:
```suggestion
authorization easy for development teams, see the **TODO** What is Oso
```

File: docs/content/any/getting-started/_index.md [hunk 8]
```diff
@@ -1,11 +1,44 @@
...
+section or **TODO** read stories from teams using Oso — like Wayfair and
```
Comment (gj) [inline docs/content/any/getting-started/_index.md:43]: are you planning to address these TODOs before shipping this PR?

File: docs/content/any/getting-started/_index.md [hunk 9]
```diff
@@ -1,11 +1,44 @@
...
+application.
+    - **Resources, Roles and Permissions**: The most common way to model
+    authorization with Oso is through resources. Specify resources in
+    Polar and the roles and permissions you want users to have on them.
+    - **Rules**: Since Polar is a declarative language, you can extend
+    the resource model to cover other cases as needed by your
+    application. For example, you may deny access by banned users or
+    allow any user to access a public resource.
+{{% ifLang "python" %}}
+- **Framework Libraries**: Oso has pre-built integrations for web
```
Comment (gj) [inline docs/content/any/getting-started/_index.md:29]: not sure what 'pre-built' means in this context

```suggestion
- **Framework Libraries**: Oso has integrations for web
```

maybe 'out-of-the-box'? although technically it's a different box (e.g., `sqlalchemy-oso` vs. `oso`)

File: docs/content/any/getting-started/_index.md [hunk 10]
```diff
@@ -1,11 +1,44 @@
...
-any: false
-below_picker: roles-cta
+any: true
 ---
 
-# Jump in
+# Get Started
 
-Learn the basics of using Oso and get Oso set up in your application.
+Let's get started with Oso! Oso is a framework for application
```
Comment (gj) [inline docs/content/any/getting-started/_index.md:9]:
```suggestion
Oso is a framework for application
```

File: docs/themes/oso-tailwind/layouts/partials/sidebar.html [hunk 11]
```diff
@@ -32,6 +32,11 @@
                 </div>
                 {{ $currentNode := . }}
                 <ul>
+                    <li class="pl-2 font-bold text-black"><a href="">
+                        <a class="flex items-center px-3 hover:text-gray-900 transition-colors duration-200" href="{{ $.Site.BaseURL }}index.html">
```
Comment (gj) [inline docs/themes/oso-tailwind/layouts/partials/sidebar.html:36]: I'm not sure what this transition is supposed to look like, but I'm not seeing much when I hover on 'Home' in the sidebar

**Review by gj — CHANGES_REQUESTED** [review]
mostly LGTM

didn't do an exhaustive check to make sure every deleted path has been added as an alias somewhere

File: docs/content/python/guides/ui/index.md (hunk 3 above)
Comment (dhatch) [reply docs/content/python/guides/ui/index.md:139]: Yeah guide is on the list for tomorrow to update.

File: docs/content/any/project/changelogs/2021-06-09.md (hunk 2 above)
Comment (dhatch) [reply docs/content/any/project/changelogs/2021-06-09.md:58]: my bad added back

File: docs/content/any/getting-started/_index.md (hunk 6 above)
Comment (dhatch) [reply docs/content/any/getting-started/_index.md:39]: Updated to "filtering collections by authorization" I don't want to make this sentence too wordy - you think that's better?

File: docs/content/any/getting-started/_index.md (hunk 8 above)
Comment (dhatch) [reply docs/content/any/getting-started/_index.md]: No I'm addressing TODOs in this entire section later today b/c they are links to stuff that's not in yet mostly.

File: docs/themes/oso-tailwind/layouts/partials/sidebar.html (hunk 11 above)
Comment (dhatch) [reply docs/themes/oso-tailwind/layouts/partials/sidebar.html:36]: I just copied styles from below

Comment (dhatch) [reply docs/themes/oso-tailwind/layouts/partials/sidebar.html:36]: removed these

**Review by gj — APPROVED** [review]
